optimizely/helpers/enums.py

```diff
@@ -115,6 +115,13 @@ class Errors(object):
     UNSUPPORTED_DATAFILE_VERSION = 'This version of the Python SDK does not support the given datafile version: "{}".'
 
 
+class ForcedDecisionNotificationTypes(object):
```

Comment:
These are not "NotificationTypes". What about "ForcedDecisionLogs"

Reply:
Done